### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by runevad**
Not at all sure if this helps, but maybe this will result in slightly fewer issues with people not getting their system up and running =)

### CODE REVIEW

File: docs/lovelace_config_example.txt [hunk 1]
```diff
@@ -0,0 +1,69 @@
+  - badges: []
+    icon: 'mdi:thermostat'
+    path: nibe_31372_0
```
Comment (elupus) [inline docs/lovelace_config_example.txt]: Not sure you want to expose your system identifier here.

File: README.md [hunk 2]
```diff
@@ -13,19 +13,59 @@ Installation
...
-cd .homeassistant
-mkdir custom_components
-cd custom_components
-git clone https://github.com/elupus/hass_nibe.git nibe
-```
+ ```bash
+ cd .homeassistant
+ mkdir custom_components
+ cd custom_components
+ git clone https://github.com/elupus/hass_nibe.git nibe
```
Comment (elupus) [inline README.md]: Think you changed line endings fro. Unix to windows.

File: README.md [hunk 3]
```diff
@@ -13,19 +13,59 @@ Installation
...
+        - system: <required system identifier>
+          units:
+            - unit: 0
+          climates: True
+          water_heaters: True
+```
+
+* Go to the Configuration tab of Home Assistant
+
+<img src="https://github.com/runevad/hass_nibe/blob/master/docs/configuration.png" alt="Configuration page" />
```
Comment (elupus) [inline README.md]: Please use relative path.

Comment (runevad) [reply README.md]: Fixed, copied wrong

File: docs/lovelace_config_example.txt (hunk 1 above)
Comment (runevad) [reply docs/lovelace_config_example.txt]: Fixed. Not sure if the lovelace example is necessary or not.

File: README.md (hunk 2 above)
Comment (runevad) [reply README.md]: Fixed. Not sure why this happened. Using standard Atom and did not touch this segments indentation.

**Comment by runevad** [comment]
Is the step entering the configuration-page overkill? Should I try to make it more compact and instead add an extra image when the configuration page asks you to enter the long generated token?

File: README.md [hunk 4]
```diff
@@ -22,10 +21,55 @@ cd custom_components
 git clone https://github.com/elupus/hass_nibe.git nibe
 ```
 
+ * If you are using windows, download the zip and extract the folder inside to your custom_components folder.
+ * Rename the folder "hass_nibe-master" to "nibe".
+ - All files should be inside the "nibe" catalog under the custom_components folder.
+
+<img src="/docs/nibe_files_windows.png" alt="Windows folder" />
```
Comment (elupus) [inline README.md]: This contain several files not in git (generated files) would be better if it was a clean checkout

File: README.md [hunk 5]
```diff
@@ -22,10 +21,55 @@ cd custom_components
...
+
+ * Add a nibe configuration block to your `<config dir>/configuration.yaml` see example below
+ - To get your system identifier log into your account at https://www.nibeuplink.com
+ - Look at the url when accessing your heat pump start page
+ - Copy the numbers that appears in the url : https://www.nibeuplink.com/System/**XXXXX**
```
Comment (elupus) [inline README.md]: The better way to find this is actually to add a empty nibe: clause to config. Then you get a notification about what systems you have.

File: README.md (hunk 4 above)
Comment (runevad) [reply README.md]: Fixed, new image uploaded

File: README.md (hunk 5 above)
Comment (runevad) [reply README.md]: Ok, first a restart with only nibe: and when the notification displays your systems you go back and add a more complete configuration entry and another restart before the entities appear on the integration page? 
or is the "richer" configuration not necessary any more?

**Comment by elupus** [comment]
Not forgotten. just much todo.

**Comment by runevad** [comment]
Don’t worry, this is one of my favorite components to my Home assistant. Let us know if we can assist in some other way.

File: README.md [hunk 6]
```diff
@@ -4,16 +4,14 @@ Nibe - An home assistant plugin to get data from Nibe Uplink
 Preparation
 ------------
 
-Register an nibe uplink application on: https://api.nibeuplink.com/
-
-  * Set the redirect url to match `<http or https>://<your_home_assistant_url_or_local_ip>:<port>/api/nibe/auth`
-  * Take note of the **Identifer** (client_id) and the **Secret**
+  - [x] Register an nibe uplink application on: https://api.nibeuplink.com/
```
Comment (elupus) [inline README.md]: I don't like the change to checkboxes. looks odd.

Comment (runevad) [reply README.md]: Ok, agree. Changed it back.

**Comment by runevad** [comment]
Not sure I used your request for changes correctly, something went wrong and I ended up editing the file here on Github instead of using Atom. But the checkboxes are gone and the text is slightly updated. 

More feedback? =)

**Comment by bohtho** [comment]
Looks good to me.

**Pull request merged**